staging/src/k8s.io/apiserver/pkg/server/filters/longrunning.go

```diff
@@ -39,3 +40,83 @@ func BasicLongRunningRequestCheck(longRunningVerbs, longRunningSubresources sets
 		return false
 	}
 }
+
+// WithLongRunningRequestTermination starts closing long running requests upon receiving TerminationStartCh signal.
```

Comment:
we have a point when the non-long-running requests are finished. That's (potentially long) after TerminationStartCh, isn't it? Do we have access to that point in time? Would be more graceful to keep long-running open longer. I understand that TerminationStartCh closes when the signal arrives?